vmdb/app/models/container_node.rb

```diff
@@ -5,16 +5,16 @@ class ContainerNode < ActiveRecord::Base
   # :name, :uid, :creation_timestamp, :resource_version
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   has_many   :container_groups
-  has_many   :container_node_conditions, :dependent => :destroy
+  has_many   :container_conditions, :class_name => ContainerCondition, :as => :container_entity, :dependent => :destroy
   has_one    :computer_system, :as => :managed_entity, :dependent => :destroy
   belongs_to :lives_on, :polymorphic => true
 
   delegate   :hardware, :to => :computer_system
 
-  virtual_column :ready_condition_status, :type => :string, :uses => :container_node_conditions
+  virtual_column :ready_condition_status, :type => :string, :uses => :container_conditions
```

Comment:
@dkorn This is interesting. Does this let you access a nodes condition from a node record directly?
What does it do if you have several conditions?